projects/angular/src/forms/datepicker/_properties.datepicker.scss

```diff
@@ -24,7 +25,7 @@
     --clr-calendar-btn-picker-trigger-hover-color: #{tokens.$cds-alias-typography-link-color-hover};
     --clr-calendar-btn-picker-trigger-active-color: #{tokens.$cds-alias-typography-link-color};
 
-    --clr-calendar-btn-hover-focus-color: #{tokens.$cds-alias-object-interaction-background-hover};
+    --clr-calendar-btn-hover-color: #{tokens.$cds-alias-object-interaction-background-hover};
```

Comment:
You can't just remove / rename already released token. Add a new token for the hover and deprecate the old ones if need while keeping the backward compatibility.